dotnet/README.md

```diff
@@ -0,0 +1,16 @@
+# Tool builder: gcr.io/cloud-builders/dotnet
+This directory contains the description for the `dotnet` Container Build step. This step will run the .NET Core `dotnet` command line. This builder has two SDKs installed:
```

Comment:
"This builder has two SDKs installed:"

This is fine, but there is precedence for having the version of the tool or SDK in the image tag. So, people would use the 'gcr.io/cloud-builders/dotnet:1.0.3' builder.

If this is a thing commonly configured through CLI flag or project config, it makes sense to have it like you do here.